lifecycle/cli.go

```diff
@@ -155,12 +155,9 @@ func create(c *cli.Context) (err error) {
 		return err
 	}
 
-	var creationSpec *spec.SpecFiles
-	if c.IsSet("spec") {
-		creationSpec, err = cliutils.GetSpec(c, true)
-		if err != nil {
-			return
-		}
+	creationSpec, err := getReleaseBundleCreationSpec(c)
```

Comment:
Instead of dealing with nil specs and dealing with flags in the core, can we create a default spec like we do in Artifactory commands?
For example - 
https://github.com/jfrog/jfrog-cli/blob/v2.56.0/artifactory/cli.go#L1188

Reply:
There isn't really a default spec here because the specs depend on the creation method. In addition, the default spec is convenient when the user can provide options instead of a spec, which isn't currently the case.